### DISCUSSION TITLE
I want use a-h/templ to write a go code template for my gogen program. How to write curly braces without them being recognized as variables?

### DISCUSSION THREAD

**Discussion by TraderWithPython** [a-h/templ#618] [Q&A]
I want use a-h/templ to write a go code template for my gogen program. How to write curly braces without them being recognized as variables?

**Comment by joerdav** [comment]
This is not recommended, templ is a HTML templating language.

**Comment by TraderWithPython** [comment]
> This is not recommended, templ is a HTML templating language.

It is indeed a very impressive package. I am just stuck on this issue right now.

**Comment by joerdav** [comment]
Sorry, what I'm trying to say is that you shouldn't write Go code templates using templ, it's for HTML.
